lib/src/callable/async.dart

```diff
@@ -23,16 +23,21 @@ abstract class AsyncCallable {
   /// The callable's name.
   String get name;
 
+  @Deprecated('Use `new AsyncCallable.function` instead.')
```

Comment:
nit: Leave out the optional `new` from the deprecation messages.

Reply:
Done.